src/config/ThemeProvider.js

```diff
@@ -34,9 +34,15 @@ export default class ThemeProvider extends React.Component {
     if (useDark !== state.useDark) {
       const defaultColors = useDark ? darkColors : colors;
       return {
-        theme: deepmerge(state.theme, {
-          colors: defaultColors,
-        }),
+        theme: deepmerge(
+          state.theme,
```

Comment:
Why is `state.theme` needed here? I think it's safer to omit it (merge default colors + props.theme).